app/src/main/java/com/socketmobile/stockCount/helper/RefersionKeys.kt

```diff
@@ -0,0 +1,26 @@
+package com.socketmobile.stockCount.helper
+
+import com.socketmobile.stockCount.model.GetAffParam
+
+val ChinaLink = "https://www.amazon.cn/dp/B01N8SQ8MU/ref=sr_1_7?__mk_zh_CN=%E4%BA%9A%E9%A9%AC%E9%80%8A%E7%BD%91%E7%AB%99&keywords=socket+mobile&qid=1555606674&s=gateway&sr=8-7"
+val EuroCodes = arrayOf("AL", "AD", "AM", "AT", "BY", "BE", "BA", "BG", "CH", "CY", "CZ", "DE", "DK", "EE", "ES", "FO", "FI", "FR", "GE", "GI", "GR", "HU", "HR", "IE", "IS", "IT", "LT", "LU", "LV", "MC", "MK", "MT", "NO", "NL", "PO", "PT")
+val DefaultKey = GetAffParam("pub_9a42760dc57269d2f616", "sec_86be30771c1af32a57df", "7daad6")
+val Keys = mapOf(
```

Comment:
Are we sure those keys need to be in source control?

Reply:
Those keys are needed to get scanner store link from refersion.